rosbag2_transport/src/rosbag2_transport/player.cpp

```diff
@@ -188,19 +195,65 @@ void Player::play_messages_until_queue_empty(const PlayOptions & options)
   }
 }
 
+bool is_package_found(rosbag2_storage::TopicMetadata topic)
```

Comment:
that name is a bit misleading in my opinion. What about `message_type_available`?